osu.Game/Screens/Ranking/ResultsScreen.cs

```diff
@@ -183,6 +195,11 @@ protected override void LoadComplete()
                 api.Queue(req);
 
             statisticsPanel.State.BindValueChanged(onStatisticsStateChanged, true);
+
+            using (BeginDelayedSequence(AccuracyCircle.ACCURACY_TRANSFORM_DELAY + AccuracyCircle.TEXT_APPEAR_DELAY, true))
+            {
+                this.Delay(-1000).Schedule(() => applauseSound?.Play());
```

Comment:
why has the number changed?

Reply:
I've prodded at this for a little bit, and while this number *does* need changing, I believe it should have changed by less, namely to

```suggestion
                this.Delay(-1140).Schedule(() => applauseSound?.Play());
```

Reasoning for this particular value is this:

https://github.com/ppy/osu/blob/41d8bc291ce81e496c660b8362567b2d9b1716a6/osu.Game/Screens/Ranking/ScorePanel.cs#L236-L237

The accuracy meter, along with the rest of the content, only starts showing after the panels have expanded. `resize_duration` is 200ms, `top_layer_expand_delay` is 100ms. Hence

```
-1440 + 200 + 100 = -1140
```

Can be roughly compared using `TestSceneResultsScreen` after setting playback speed low (~15% is best). The audio should start playing very soon after the accuracy meter appears and starts filling. Seemed more correct, can make videos of this if necessary.